docs/src/release_notes/old/v1.15.md

```diff
@@ -211,7 +211,7 @@ Features:
 - Support for the `AWS_SESSION_TOKEN` authentication token in AWS S3 through
   the `sessionToken` option
 - Default image name for PgBouncer in `Pooler` pods set to
-  `quay.io/enterprisedb/pgbouncer:1.17.0`
```

Comment:
Please do not change the old documentation, it's been kept like that for historical reasons =)